src/login/Login.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Redirect } from "react-router-dom";
+import "../myspace/MySpace";
+
+
+export default function Login() {
+    const [errorMessage, setResponseMessage] = useState(null);
```

Comment:
errorMessage, setErrorMessage - try following the convention

Reply:
yep, missed that 😁